ozaria/engine/cinematic/CinematicLankBoss.js

```diff
@@ -77,28 +77,28 @@ export default class CinematicLankBoss {
     }
 
     const lHero = getLeftHero(shot)
-    const original = me.get('heroConfig').thangType
+    const original = (me.get('heroConfig') || {}).thangType || '55527eb0b8abf4ba1fe9a107'
```

Comment:
Lets set `55527eb0b8abf4ba1fe9a107` to a constant at the top of this module so that we understand what it is.